pkg/controllers/disruption/controller.go

```diff
@@ -132,6 +132,9 @@ func (c *Controller) Reconcile(ctx context.Context) (reconcile.Result, error) {
 	if err := state.RequireNoScheduleTaint(ctx, c.kubeClient, false, lo.Filter(c.cluster.Nodes(), func(s *state.StateNode, _ int) bool {
```

Comment:
Can you update the comment here to change from "If Karpenter restarts during a disruption action", since now we're relying on this every time we fail an action, even if Karpenter hasn't restarted?